static/docs/command-reference/config.md

```diff
@@ -55,7 +55,7 @@ corresponding config file.
 
 ## Configuration sections
 
-The following config sections are written by this command to the project config 
+The following config sections are written by this command to the project config
```

Comment:
This is what I was talking about in https://github.com/iterative/dvc.org/issues/861 . I didn't even touch this, but hooks fixed this anyways.

Reply:
It look like a problem that master is "green" but we have a bug in the formatting cc @jorgeorpinel

Reply:
My hook recently also fixed this particular file. I committed this fix in f36b098 so no t sure why it even shows this as a difference from `master`.

Reply:
p.s. Shown here in blame in the current `master` version: https://github.com/iterative/dvc.org/blame/master/static/docs/command-reference/config.md#L58

Reply:
> look like a problem that master is "green" but we have a bug in the formatting

`yarn format-check` used in CI only checks changed files.

Reply:
This one broke in 488fccd. Not sure how this got merged.

Reply:
🤦‍♂ looks like I merged a bad PR https://github.com/iterative/dvc.org/pull/840. I didn't realize GH won't stop you from merging a PR with failing checks 😅 (https://circleci.com/gh/iterative/dvc.org/1556)

Reply:
Seems like this is no longer a diff. Resolving. We'll fix the formatting woes though, don't worry.